docs/internals/contributing/committing-code.txt

```diff
@@ -132,18 +133,18 @@ Django's Git repository:
   lines. The limits are soft. For the subject line, shorter is better. In the
   body of the commit message more detail is better than less::
 
-      Fixed #18307 -- Added git workflow guidelines
+      Fix #18307 -- Add git workflow guidelines
 
-      Refactored the Django's documentation to remove mentions of SVN
```

Comment:
"the Django's documentation" -> "Django's documentation" (I know this is unrelated, just may as well fix it while here?